patches/github-auth.diff

```diff
@@ -5,24 +5,11 @@ Microsoft's does not work with self-hosted instances so we run our own.
 Also add an extra set of scopes so that tokens provided via --github-auth will
 work for the PR extension.
 
-Index: code-server/lib/vscode/extensions/github-authentication/src/githubServer.ts
-===================================================================
---- code-server.orig/lib/vscode/extensions/github-authentication/src/githubServer.ts
-+++ code-server/lib/vscode/extensions/github-authentication/src/githubServer.ts
-@@ -17,7 +17,7 @@ const localize = nls.loadMessageBundle()
- const CLIENT_ID = '01ab8ac9400c4e429b23';
- 
- const NETWORK_ERROR = 'network error';
--const AUTH_RELAY_SERVER = 'vscode-auth.github.com';
-+const AUTH_RELAY_SERVER = 'auth.code-server.dev';
- // const AUTH_RELAY_STAGING_SERVER = 'client-auth-staging-14a768b.herokuapp.com';
- 
- class UriEventHandler extends vscode.EventEmitter<vscode.Uri> implements vscode.UriHandler {
```

Comment:
This is my biggest concern for 1.66. I haven't tested this locally yet but I assume this will completely break our GitHub auth flow.

Reply:
I was testing GitHub auth this morning on 4.3.0 and instead of doing the regular redirect auth flow it asked me to click on a link and input a code so it is possible this code is no longer even used on web and we can just ignore it.

Reply:
OMG! I really hope so that. That would make our lives so much easier (++patches removed too 🔥)

Reply:
Yes!  Exactly my thoughts.

Reply:
<img width="408" alt="image" src="https://user-images.githubusercontent.com/3806031/165829687-68d32751-d219-49c0-a04b-8e72727d18b1.png">

I tried before deleting the patch and it worked!

I wonder about deleting the patch though since you had to modify those scopes  🤔I wonder if that logic even works still (as in, is used). I'll have to check.

Reply:
Ooo good point about the scopes, we might still need that to support the `--github-auth` flag

Reply:
I removed the editor and it appears that everything in the PR extension is working 

https://github.com/coder/code-server/pull/5141